infra/examples-dev/aws/az-auth

```diff
@@ -0,0 +1 @@
+../../../tools/az-auth.sh
```

Comment:
is this a symlink??? looks like, the icon is different. Then use it in `build` too